rpm/SPECS/docker-ce-cli.spec

```diff
@@ -18,7 +18,12 @@ Requires: /bin/sh
 Requires: /usr/sbin/groupadd
 # TODO change once we support scan-plugin on other architectures
 %ifarch x86_64
+# CentOS 7 and RHEL 7 do not yet support weak dependencies
+%if 0%{?rhel} == 7
```

Comment:
note that I'm specifically _not_ using `<` or `>`  here, as `rhel` won't be / may not be defined on  other RPM platforms, in which case `0` (default) is picked

Reply:
^^ added a note about that in the comment to prevent someone accidentally changing it

Reply:
I only hope this will always be `7` and not `7.1` (e.g.) on CentOS / RHEL 🤔